alicloud/resource_alicloud_log_dashboard.go

```diff
@@ -38,8 +41,9 @@ func resourceAlicloudLogDashboard() *schema.Resource {
 				Optional: true,
 			},
 			"char_list": {
-				Type:     schema.TypeString,
-				Required: true,
+				Type:             schema.TypeString,
+				DiffSuppressFunc: jsonPolicyDiffSuppress,
```

Comment:
There is a stable diffsuppressfunc to support it and please refer to: https://github.com/aliyun/terraform-provider-alicloud/blob/master/alicloud/resource_alicloud_ram_policy.go#L67

Reply:
`compareJsonTemplateAreEquivalent` can't work here. In my testing, `terrarform apply` show changes every time. 

Before comparing the chart lists, we must sort the charts.

Reply:
I guess the alicloud API reordered the chart list, so after the configurations are applied to the alicloud,  the`chart_list` may diff from the user configuration in the next `terraform apply` (using compareJsonTemplateAreEquivalent). 

However, the order of the chart list does not matter. I think it's better to sort the charts before comparing in this case.